crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -72,6 +72,39 @@ impl WrapEvtSender {
         )
     }
 
+    fn query_gossip_agents(
+        &self,
+        dna_hash: DnaHash,
+        to_agent: AgentPubKey,
+        kitsune_space: Arc<kitsune_p2p::KitsuneSpace>,
+        since_ms: u64,
+        until_ms: u64,
+        arc_set: Arc<kitsune_p2p_types::dht_arc::DhtArcSet>,
```

Comment:
I was scanning this code and it occurred to me that there's quite a surprising overlap between names coming up in the code here, given Atomic Reference Counting = Arc, and then the DHT related implementation is dealing in Arc's. The only thing that came to my mind about this is possibly leaving simple and short code comments to distinguish that the Arc<> on the outside is different than the ArcSet on the inside for future code reviewers and Holochain contributors?

Reply:
Perhaps. I mean DhtArc is hopefully clearly different to Arc. Where would you put the comment though? This type is used in a lot of places. I hope that anyone familiar enough with rust would see Arc and know what that is. The only other thing to do would be to change the name from `DhtArc`?

Reply:
Yeah, there's really nothing to be done for it thats useful. I suppose I just thought it was amusing in a way, and unlikely, and pictured some poor reader down the road :p nvm me

Reply:
We could call it a `Darc` for short?

Reply:
The distinction seems clear to me in code, but then when we talk about it more colloquially and in comments, there is this unfortunate name collision.

I still think some variant of "arc" is the best name for it, since it is really helpful to think of it as an arc of a circle, and that is the correct geometrical term.

The only other geometrical term that could make sense is a "sector", which includes the arc points but also includes the area between the two radii. This seems less precise, since really we are only dealing with the one-dimensional wrapping space of DHT locations, and I don't know what the area inside the circle would represent.